src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -174,6 +174,17 @@ void maybeInitializePreprocessorSymbolTable(AbstractCompiler compiler) {
       }
     }
   }
+  
+  @Override
+  protected List<PassFactory> getAlwaysPreChecks() {
+    List<PassFactory> preChecks = new ArrayList<>();
+    
+    if (options.wrapGoogModulesForWhitespaceOnly) {
```

Comment:
Happy to simplify, remove the option, and tie it to whitespace + closurePass mode. I only put it there to make it unambiguously an opt-in change given the debatable nature of doing changes in a so-called whitespace-only mode ;-)